proj/src/main/webapp/chat.js

```diff
@@ -526,6 +536,9 @@ function createMessageWithTemplate(key, messageObj) {
             message.querySelector('#pfp').src = src;
         })
     })
+    if(messageObj.timestamp > lastVisited && messageObj.senderUID != firebase.auth().currentUser.uid){
+        message.querySelector('.unread').innerText = " (New)";
```

Comment:
This is likely to be more useful to Lee or whoever cleans it up, but I think the standard way to do this is to have a CSS class that describes how you want to display unread messages, and then just apply that class to these messages.  That way, the styling can be changed without touching the code.

Reply:
Maybe just a slightly different background color for unread messages as opposed to adding text?

Reply:
I agree that's probably the better stylistic approach, but the beautiful thing about applying a class is that you can then do either just by changing some CSS.